<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoInput</class>
 <widget class="QWidget" name="VideoInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoInput</string>
  </property>
  <widget class="QLabel" name="videoLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>471</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string>videoData</string>
   </property>
  </widget>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>391</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string>photoData</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="playBtn">
      <property name="text">
       <string>播放</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="photoBtn">
      <property name="text">
       <string>拍照</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopBtn">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
